src/lib/i18n/dictionaries/dictionary.json

```diff
@@ -4438,6 +4438,26 @@
   "npcDialogues.corale.chore1": "The coral reefs are buzzing with news about strange sea creatures!",
   "npcDialogues.corale.chore2": "Something big is disturbing the ocean's natural balance...",
   "npcDialogues.corale.chore3": "The fish say they've seen cow-like shadows in deeper waters!",
+
+  "npcDialogues.santa.intro1": "Ho Ho Ho Bumpkin! For a limited time, I am giving daily rewards for those that find my missing Candy.",
+  "npcDialogues.santa.intro2": "Each day you will find 10 pieces scattered throughout the Pumpkin Plaza.",
+  "npcDialogues.santa.intro3": "Collect these pieces 12 days in a row for a special prize!",
```

Comment:
We often get confusion around these events if players can 'miss' a day.

I would suggest the following text: "Collect the candy and claim the daily prizes"

Reply:
<img width="554" alt="Screenshot 2024-12-06 at 9 06 08 AM" src="https://github.com/user-attachments/assets/c9295bbd-0316-4c2d-9e03-207891eca975">

I would consider adding an extra blue label - "18 days left" which displays how many days left in the event 👍